### DISCUSSION TITLE
ui.command elements in selected rows

### DISCUSSION THREAD

**Discussion by shamilh2o** [h2oai/wave#1942] [Q&A] [answered]
Hi team,

Can we add ui.command elements only for selected rows of a table?
Previously I got to know that was a limitation in wave. Is that addressed in a recent release? 

Thanks in advanced.

**Comment by mturoci** [comment]
Hi, this is not possible. You can only add a command menu for either all or nothing.

**Reply by shamilh2o** [reply]
Hey. Thanks for the update @mturoci . 
Any plan of addressing this in an upcoming release? Or shall I create a FR on this?

**Reply by mturoci** [reply, marked as answer]
No plans. Not even sure it's possible to add. However, I may consider merging a PR if provided.

**Reply by shamilh2o** [reply]
Thank you will create a FR on this.
